src/components/Survey/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export const AnythingWrongSurvey = () => {
+  const { t } = useTranslation();
+  return (
+    <div className="margin-top-4">
```

Comment:
Looks like there's an extra `div` here. This class can probably go directly on the `p`.